src/Box/Box.ts

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+import {
+  borders,
+  BordersProps,
+  boxShadow,
+  BoxShadowProps,
+  display,
+  DisplayProps,
+  gridArea,
+  GridAreaProps,
+  height,
+  HeightProps,
+  maxHeight,
+  minHeight,
+  position,
+  PositionProps,
+  width,
+  WidthProps
+} from 'styled-system';
+
+import Text, { TextProps } from '../Text';
+
+export type Props = TextProps &
+  BordersProps &
+  BoxShadowProps &
+  DisplayProps &
+  HeightProps &
+  GridAreaProps &
+  PositionProps &
+  WidthProps;
```

Comment:
Fortunately, `Props` for things like this can be composed almost entirely of props defined by `styled-system`.